### ISSUE TITLE
[3.6.0] Focus ring for dropdown item missing

### ISSUE THREAD

**Issue by distantnative** [getkirby/kirby#3355]
https://github.com/getkirby/kirby/blob/develop/panel/src/components/Navigation/DropdownItem.vue#L64

**Comment by distantnative** [comment]
✅
